ksp_plugin/pile_up.cpp

```diff
@@ -419,44 +426,153 @@ void PileUp::DeformPileUpIfNeeded(Instant const& t) {
   // is unaffected by the apparent-bubble-to-pile-up correction, which is rigid
   // and involves no change in axes.
   auto const inertia_tensor = apparent_system.InertiaTensor();
+
+  // This is the axis around which we may perform an orientation correction.
+  // Identifying the uncorrected (apparent) and corrected coordinates, it is
+  // orthogonal to both.
+  // Note that the computations are identical in coordinates:
+  // |apparent_correction_axis.coordinates() ==
+  //  actual_correction_axis.coordinates()|.
+  auto const apparent_correction_axis = NormalizeOrZero(Commutator(
+      apparent_angular_momentum,
+      Identity<NonRotatingPileUp, ApparentPileUp>()(angular_momentum_)));
+  auto const actual_correction_axis = NormalizeOrZero(Commutator(
+      Identity<ApparentPileUp, NonRotatingPileUp>()(apparent_angular_momentum),
+      angular_momentum_));
+
+  // We apply a rigid rotational correction to the motions of the parts coming
+  // from the game (the apparent motions) so as to enforce the conservation of
+  // the angular momentum (|angular_momentum_| is authoritative).
+  // Mapping L_apparent to L_actual by a rigid motion can be done in many ways.
```

Comment:
Should the `L`s have little hats?

Reply:
Not these. The rigid motion includes the angular velocity, so it maps the bareheaded angular momenta to each other.